ocs_ci/ocs/resources/pod.py

```diff
@@ -573,32 +574,40 @@ def get_fio_rw_iops(pod_obj):
     )
 
 
-def run_io_in_bg(pod_obj, expect_to_fail=False):
+def run_io_in_bg(pod_obj, expect_to_fail=False, fedora_dc=None):
     """
     Run I/O in the background
 
     Args:
         pod_obj (Pod): The object of the pod
         expect_to_fail (bool): True for the command to be expected to fail
             (disruptive operations), False otherwise
+        fedora_dc(str): set to None by default. If set to True, it runs IO in
+            background on a fedora dc pod.
 
     Returns:
         Thread: A thread of the I/O execution
     """
     logger.info(f"Running I/O on pod {pod_obj.name}")
 
-    def exec_run_io_cmd(pod_obj, expect_to_fail):
+    def exec_run_io_cmd(pod_obj, expect_to_fail, fedora_dc):
         """
         Execute I/O
         """
         try:
             # Writing content to a new file every 0.01 seconds.
             # Without sleep, the device will run out of space very quickly -
             # 5-10 seconds for a 5GB device
-            pod_obj.exec_cmd_on_pod(
-                f"bash -c \"let i=0; while true; do echo {TEXT_CONTENT} "
-                f">> {TEST_FILE}$i; let i++; sleep 0.01; done\""
-            )
+            if fedora_dc:
```

Comment:
instead of calling pod_obj.exec_cmd_on_pod twice, please call it once with the relevant test file which will be determined prior to this function call, according to fedora_dc attribute

Reply:
Done